src/pages/Favorites/Favorites.page.jsx

```diff
@@ -1,11 +1,31 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import _ from 'underscore';
 
+import VideoList from '../../components/VideoList';
+import { FAV_STORAGE_KEY } from '../../utils/constants';
+import { storage } from '../../utils/storage';
 import { Container, Title } from './Favorites.styles';
 
 const FavoritesPage = () => {
-  return (
-    <Container>
-      <Title data-testid="favorites-message">Favorites</Title>
+  const [videos, setVideos] = useState([]);
+
+  useEffect(() => {
+    const myFavs = storage.get(FAV_STORAGE_KEY);
+    if (myFavs) {
+      setVideos(_.flatten(_.values(myFavs)));
+    } else {
+      setVideos(null);
```

Comment:
the default value for videos at ln 10 is an empty array, here it is set to null with the risk of failing at ln 21 due to try to reach a length prop of an undefined value.  So I suggest to set it to an empty array instead.

Even better, since this effect is going to be executed at first render only and you have a default value at ln 10, you could get rid of the else case.